crypto/src/mls/conversation/pending_conversation.rs

```diff
@@ -28,11 +28,14 @@ use tls_codec::Deserialize as _;
 pub struct PendingConversation {
     inner: PersistedMlsPendingGroup,
     context: TransactionContext,
+    // This is ridiculous.
+    conversation_id: ConversationId,
```

Comment:
Why add this field to the struct instead of continuing to defer to `self.inner.id`?

Reply:
I think the reason why Ivan did that is that the getter `id()` returns `self.inner.id` as `ConversationId`. But `self.inner.id ` lives in the keystore and is a `Vec<u8>`. So now the getter would need to create a ConversationId which involves cloning the Vec. 
His approach is to safe the ConversationId directly on the struct so that getting the id as the proper type is cheap.